tests/sequence_sharding_optimistic/run.sh

```diff
@@ -47,6 +42,20 @@ run() {
     # use sync_diff_inspector to check schema fetching increment data.
     check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
+    # check database `sharding_seq_tmp` exists
+    run_sql "select count(*) from sharding_seq_tmp.t1;" $TIDB_PORT $TIDB_PASSWORD
+    check_contains "count(*): 1"
+
+    # now, for optimistic shard DDL, different sources will reach a stage often not at the same time,
+    # in order to simply the check and resume flow, only enable the failpoint for one DM-worker.
+    export GO_FAILPOINTS="github.com/pingcap/dm/syncer/FlushCheckpointStage=return(100)" # for all stages
+    echo "restart dm-worker1"
+    ps aux | grep dm-worker1 |awk '{print $2}'|xargs kill || true
+    check_port_offline $WORKER1_PORT 20
+    run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
+    check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT
+    export GO_FAILPOINTS=''
+
```

Comment:
😕 can not understand these code

Reply:
😕 plus one

Reply:
enable failpoint for one DM-worker (before start the process) and then disable the failpoint (after the process started) to avoid affecting other processes in this shell session.

Reply:
> enable failpoint for one DM-worker (before start the process) and then disable the failpoint (after the process started) to avoid affecting other processes in this shell session.

Why is `maxStage` set to 100?

Reply:
just a value greater than all used stages, may a greater value like MaxInt32?